tests/functional/fx_firstrun_v1_sign_up.js

```diff
@@ -14,6 +14,11 @@ define([
   var email;
   const PASSWORD = '12345678';
 
+  const SELECTOR_CONFIRM_HEADER = '#fxa-confirm-header';
+  const SELECTOR_CONNECT_ANOTHER_DEVICE_HEADER = '#fxa-connect-another-device-header';
+  const SELECTOR_SIGN_UP_HEADER = '#fxa-signup-header';
+  const SELECTOR_SIGN_UP_SUB_HEADER = '#fxa-signup-header .service';
```

Comment:
Given that we define these selectors in more than one test module, maybe it will save us future maintenance burden to stick them in a dedicated constants module and then have one place to update them if they change in the future?